app/services/podcasts/get_media_url.rb

```diff
@@ -0,0 +1,41 @@
+module Podcasts
+  class GetMediaUrl
+    def initialize(enclosure_url)
+      @enclosure_url = enclosure_url
+    end
+
+    def self.call(*args)
+      new(*args).call
+    end
+
+    def call
+      was_http = !enclosure_url.starts_with?("https")
```

Comment:
is there a remote possibility that the URL is stored with the wrong case? "HTTPS://example.com" works in the browser